src/foundry/client/canvas/sources/base-effect-source.d.mts

```diff
@@ -138,16 +145,18 @@ declare abstract class BaseEffectSource<
    * @param data    - Provided data for configuration
    * @param options - Additional options which modify source initialization
    * @returns The initialized source
+   * @privateRemarks This should be returning `this`, but allowing the Initialized overrides requires returning a specific class
    */
-  // options: not null (destructured)
-  initialize(data?: NullishProps<SourceData> | null, options?: BaseEffectSource.InitializeOptions): this;
+  initialize(
+    data?: InexactPartial<SourceData>,
```

Comment:
Yeah idk I never love stuff like this inline but I don't see a great alternative so ah well.